client/components/TestQueue/index.jsx

```diff
@@ -52,21 +52,25 @@ class TestQueue extends Component {
                         </tr>
                     </thead>
                     <tbody>
-                        {runIds.map(runId => (
-                            <TestQueueRun
-                                key={runId}
-                                runId={runId}
-                                apgExampleName={
-                                    cycle.runsById[runId].apg_example_name
-                                }
-                                testers={cycle.runsById[runId].testers}
-                                cycleId={cycle.id}
-                                usersById={usersById}
-                                userId={userId}
-                                atName={at_name}
-                                browserName={browser_name}
-                            />
-                        ))}
+                        {runIds.map(runId => {
+                            let testsForRun = testsForRuns[runId].tests;
```

Comment:
You can remove this and just use `testsForRuns[runId].tests` below

Reply:
very reasonable